provider/pkg/test-programs/simple/index.ts

```diff
@@ -0,0 +1,56 @@
+import * as apigateway from "@pulumi/aws-apigateway";
+import * as aws from "@pulumi/aws";
+
+import * as pulumi from "@pulumi/pulumi";
+
+const config = new pulumi.Config()
+
+const useSwaggerSpec = config.getBoolean("useSwaggerSpec") || true;
+const useBinaryMediaType = config.getBoolean("useBinaryMediaType") || true;
```

Comment:
Defaulting these to true must mean that we never actually test the false cases. Can we change these to false?